_data/program_2021.yml

```
@@ -0,0 +1,48 @@
+- date: "2021-03-24"
+  dateReadable: "Day 1: Wednesday, 24<sup>th</sup> March 2021"
+  tracks:
+    - { title: "TrainTrack", color: "#90be4e" }
+    - { title: "HackTrack", color: "#03a9f4" }
+  timeSlots:
+    - {
+        startTime: "08:00",
+        endTime: "09:00",
+        sessionTitle: "coffee break",
+        speaker,
+        sessionType,
+        slides,
+      }
+    - {
+        startTime: "09:00",
+        endTime: "10:00",
+        sessionTitle: "Best session event",
+        speaker: "Marie Curie",
+        sessionType,
+        slides: "google.com",
+      }
+
+- date: "2021-03-25"
+  dateReadable: "Day 2: 25th March 2021"
+  tracks:
+    - { title: "TrainTrack", color: "#90be4e" }
+    - { title: "HackTrack", color: "#03a9f4" }
+  timeSlots:
+    - { startTime: "08:00", endTime: "09:00", sessionTitle: "coffee break" }
+    - {
+        startTime: "09:00",
+        endTime: "10:00",
+        sessionTitle: "Best session event",
+      }
+
+- date: "2021-03-26"
+  dateReadable: "Day 2: 26th March 2021"
+  tracks:
+    - { title: "TrainTrack", color: "#90be4e" }
+    - { title: "HackTrack", color: "#03a9f4" }
+  timeSlots:
+    - { startTime: "08:00", endTime: "09:00", sessionTitle: "coffee break" }
```

Ok I changed my mind, I think it would be good to already have the timeslots in there as people may already want to see at what times OpenMR will be (since we will be sending out a save the date with a link to this page). Here we go (this is most likely not properly yml formatted, so sorry abt that, just learning how to add suggestions haha):

```suggestion
    - { startTime: "10:00", endTime: "16:00", speaker: "TBD", sessionTitle: "TBD",  sessionType: "TBD", slides, }
```